resources/js/callbacks.js

```diff
@@ -1,8 +1,7 @@
 import 'Vendor/rapidez/core/resources/js/vue'
 
 Vue.prototype.transformAmCustomFormResponse = async function (data, response) {
-    let data
-    if(!(data = response.data.data.customform)) {
```

Comment:
is `data.data` still correct in rapidez v2?
i'm expecting it'll always return null

Reply:
I don't know **yet**. I'm gonna test this later. Keep this PR open @royduin